app/models/fullcalendar_engine/event_series.rb

```diff
@@ -1,7 +1,7 @@
 module FullcalendarEngine
   class EventSeries < ActiveRecord::Base
     
-    attr_accessor :title, :description, :commit_button
+    attr_accessor :title, :description, :commit_button, :color
```

Comment:
title and description are added as accessors in event_series model so that they can be used to set the values for any event model using that value. See [here](https://github.com/vinsol/fullcalendar-rails-engine/blob/master/app/models/fullcalendar_engine/event_series.rb#L20)
Since I don't see that you are setting the color of the event any where in the code, that's why I asked you to remove it.